src/editors/containers/ProblemEditor/components/SelectTypeModal/content/ProblemTypeSelect.tsx

```diff
@@ -1,24 +1,35 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Button, Container } from '@openedx/paragon';
-import { FormattedMessage, injectIntl } from '@edx/frontend-platform/i18n';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
 
 // SelectableBox in paragon has a bug where you can't change selection. So we override it
 import SelectableBox from '../../../../../sharedComponents/SelectableBox';
-import { ProblemTypes, ProblemTypeKeys, AdvanceProblemKeys } from '../../../../../data/constants/problem';
+import {
+  ProblemTypes,
+  ProblemTypeKeys,
+  AdvanceProblemKeys,
+  AdvancedProblemType,
+  ProblemType,
+} from '../../../../../data/constants/problem';
 import messages from './messages';
 
-const ProblemTypeSelect = ({
+interface Props {
+  selected: ProblemType;
+  setSelected: (selected: ProblemType | AdvancedProblemType) => void;
+}
+
+const ProblemTypeSelect: React.FC<Props> = ({
   selected,
   setSelected,
 }) => {
   const handleChange = e => setSelected(e.target.value);
   const handleClick = () => setSelected(AdvanceProblemKeys.BLANK);
-  const settings = { 'aria-label': 'checkbox', type: 'radio' };
+  const settings = { type: 'radio' };
```

Comment:
a11y fix: in the "problem types" menu, every option had the same label for screen readers: 'checkbox'. This was worse than no label at all. Now they are labelled by their inner text ("Multiple choice", "Single select", etc.)